app/src/main/java/com/github/houseorganizer/houseorganizer/house/HouseSelectionActivity.java

```diff
@@ -137,9 +202,21 @@ protected void onStop() {
         adapter.stopListening();
     }
 
+    @Override
+    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
```

Comment:
I am sorry but I don't understand what this function do ? Does it simply show if the permission is not granted ?

Reply:
Yes, it informs the user wether the requested permission has been given or not